private/private.go

```diff
@@ -12,12 +12,13 @@ type PrivateTransactionManager interface {
 	Receive(data []byte) ([]byte, error)
 }
 
-func FromEnvironmentOrNil(name string) PrivateTransactionManager {
+func FromEnvironment(name string) PrivateTransactionManager {
 	cfgPath := os.Getenv(name)
 	if cfgPath == "" {
-		return nil
+		//no privacy manager specified, start in public-only mode
```

Comment:
We don't want to default it to public but we want the user to specify whether to accept private transactions or not, hence we need PRIVATE_CONFIG to be configured.

Reply:
Alright, I'll add a check to the startup procedure to exit if the variable isn't defined, much like the consensus check.